TemplePlus/dispatcher.cpp

```diff
@@ -19,6 +19,8 @@ class DispatcherReplacements : public TempleFix {
 public:
 
 	static int Dispatch54AoE(objHndl, DispIO *, D20DispatcherKey);
+	static int GetNumNaturalAttacks_(objHndl handle);
```

Comment:
Why is this in dispatcher.cpp?? It has nothing to do with dispatchers...

Usually you can use the raw address to find a better place for it.

In this case it's obviously part of critterSys.